src/utils/buffer_reader.rs

```diff
@@ -165,4 +165,13 @@ impl<'a> BuffReader<'a> {
         }
         return &self.buffer[self.position..total_len];
     }
+
+    /// Peeking (without incremental internal pointer) one byte from buffer as `Big endian`
```

Comment:
What do you mean by big endian in this context?

Reply:
This is copy&paste from `read_u8` comment, just with added the comment about not incrementing the pointer. I guess all methods have this comment and for other integer sizes it makes sense and here it is just for consistency or copy&paste.